omod/src/main/webapp/reports/manageReports.jsp

```diff
@@ -82,7 +82,7 @@
 							&nbsp;
 							<a href="${editUrl}"><img src="<c:url value='/images/edit.gif'/>" border="0"/></a>
 							&nbsp;
-							<a href="javascript:confirmDelete('${reportDefinition.name}','${reportDefinition.uuid}');"><img src="<c:url value='/images/trash.gif'/>" border="0"/></a>
+							<a href="javascript:confirmDelete('<c:out value="${openmrs:getSafeJsString(reportDefinition.name)}" />','${reportDefinition.uuid}');"><img src="<c:url value='/images/trash.gif'/>" border="0"/></a>
```

Comment:
Yes. We have inline javascript embedded into html here, so we have to make sure the input is safe for use in both languages. The problem with c:out alone is that it converts all single/double quote characters to html entities, which break the in-line javascript.